### DISCUSSION TITLE
how to solve this problem

### DISCUSSION THREAD

**Discussion by AviDevs31** [domcloud/domcloud#63] [General]
Function symlink enabled
This function can be enabled in the php.ini file (/etc/php.ini), by editing the value of disable_functions. On a shared web hosting, you may have to ask their support.

**Comment by willnode** [comment]
All PHP functions are enabled, you can read here if you need to change values in php.ini
https://domcloud.co/docs/deployment/php#php-ini-configuration

**Reply by AviDevs31** [reply]
_**greetings sorry it was not that error I was wrong is this that gives me if you want to enter my web URLs 
rewrite enabled
You can follow the instructions in our documentation to enable URL rewriting.

By the way I'm Cristofer from telegram who helped you to translate the webs to spanish, I don't have wifi on the other pc and I had been damaged :C**_

**Comment by AviDevs31** [comment]
greetings sorry it was not that error I was wrong is this that gives me if you want to enter my web URLs
rewrite enabled
You can follow the instructions in our documentation to enable URL rewriting.

By the way I'm Cristofer from telegram who helped you to translate the webs to spanish, I don't have wifi on the other pc and I had been damaged :C
